src/stream/src/executor/mview/materialize.rs

```diff
@@ -110,7 +109,7 @@ impl<S: StateStore, SD: ValueRowSerde> MaterializeExecutor<S, SD> {
             info: ExecutorInfo {
                 schema,
                 pk_indices: arrange_columns,
-                identity: format!("MaterializeExecutor {:X}", executor_id),
```

Comment:
putting executor_id in the identity seems to be a convention for all streaming executors. I suspect this change will not only affect grafana metrics but also affect other debugging tools like async stack trace. I think it is better not changing it for materialized executor.

cc @lmatz @BugenZhao

Reply:
Yes, this is used in the span name of await-tree. But the actor id and the operator id will still be included.

https://github.com/risingwavelabs/risingwave/blob/004cc462a8e2c8763606bda3cc47dc250efcf3ac/src/stream/src/executor/wrapper/trace.rs#L75-L82